writting_functions.py

```diff
@@ -0,0 +1,99 @@
+def write_music(fl_melody, midi_file):
+    """
+    this function takes all melody flows and writes them into file one after another
+    if first number is 0 -> it is common melody, 1 -> accord-melody
+    :param fl_melody: (list of melodies)
```

Comment:
what's `fl`? =)

Reply:
fixed, it was inappropriate reduction of "full"